### PR TITLE
Upgrade sentry and enable tracing

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We want to enable performance monitoring
- We want to upgrade sentry

## This pull request

- Upgrades sentry to the latest version
- Enables tracing by setting the tracesSampleRate to 1 by default.
- Tracing can be altered / disabled by setting SENTRY_TRACES_SAMPLE_RATE to a value between 0 and 1.

## Issue that this pull request solves

Closes: FXA-8219, FXA-8103

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)
image

## Other information (Optional)

Note that currently only the front ends have instrumented cleanly out of the box. We will need to file a follow up ticket for getting tracing (which drives performance monitoring) working on the server side. It's important to know that auto instrumentation is not supported for hapi (see this [issue](https://github.com/getsentry/sentry-javascript/issues/2172)), and that it seems like nestjs is in a similar state. 

Interestingly enough Sentry [supports otel](https://develop.sentry.dev/sdk/performance/opentelemetry/) and it appears to be the [recommended approach](https://github.com/getsentry/sentry-javascript/discussions/6973) in some cases. Since we support otel instrumentation on the server side already, we will look into taking this approach. A short experiment with auth server was promising, but definitely deserves its own PR, since it requires upgrading otel and some other tweaks as well.

For now, we will apply these changes, so we can get some client side performance metrics for content server despite knowing the tracing capabilities will be incomplete for server side requests.

### CODE REVIEW

File: packages/fxa-admin-panel/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Outside of local development, this application is protected by SSO, a VPN connec
 
 This service will automatically spin up when `yarn start` is [ran from the root directory](https://github.com/mozilla/fxa#getting-started). A small Express server serves a React application and exposes the server config file for the client to consume via a meta tag.
 
-The React dev server runs at [localhost:8092](http://localhost:8092/) which can be useful when building components if you'd like an auto page refresh on file changes, however, the Express server that serves the React application and proxies its static resources runs at [localhost:8091](http://localhost:8091/). Develop on `:8091` if you need access to anything set in the server configuration file, including the URI for connecting to the `fxa-admin-server`.
+The React dev server runs at [localhost:8091](http://localhost:8092/) which can be useful when building components if you'd like an auto page refresh on file changes, however, the Express server that serves the React application and proxies its static resources runs at [localhost:8091](http://localhost:8091/). Develop on `:8091` if you need access to anything set in the server configuration file, including the URI for connecting to the `fxa-admin-server`.
```
Comment (chenba) [inline packages/fxa-admin-panel/README.md:17]: The port number in the link need to be updated to match?

File: packages/fxa-auth-server/lib/google-maps-services.ts [hunk 2]
```diff
@@ -49,7 +50,7 @@ export class GoogleMapsService {
         scope.setContext('googleMapsService', {
           address,
         });
-        Sentry.captureMessage(error.message, Sentry.Severity.Error);
+        Sentry.captureMessage(error.message, 'error' as SeverityLevel);
```
Comment (chenba) [inline packages/fxa-auth-server/lib/google-maps-services.ts:53]: Sentry removed the severity enum in the new version?

**Review by chenba — APPROVED** [review]
👍   Thanks for updating those calls, Dan

File: packages/fxa-admin-panel/README.md (hunk 1 above)
Comment (dschom) [reply packages/fxa-admin-panel/README.md:17]: I was having some trouble with startup and csp blocking things on admin panel. This was just cruft from trying to get that working better. I'll revert this change. I might revisit this and file another small PR.

File: packages/fxa-auth-server/lib/google-maps-services.ts (hunk 2 above)
Comment (dschom) [reply packages/fxa-auth-server/lib/google-maps-services.ts:53]: They did! I was also surprised by this. See release notes [here](https://github.com/getsentry/sentry-javascript/blob/develop/MIGRATION.md#severity-severitylevel-and-severitylevels).

**Pull request merged**